network-api/networkapi/wagtailpages/pagemodels/blog/mock_search_blog_index.py

```diff
@@ -0,0 +1,29 @@
+from wagtail.core.models import Page
+from wagtail.contrib.routable_page.models import RoutablePageMixin, route
+from networkapi.wagtailpages.utils import get_locale_from_request, get_default_locale
+from .blog import BlogPage
+
+
+class MockSearchBlogIndex(RoutablePageMixin, Page):
+
+    template = 'wagtailpages/mock_search_blog_index.html'
+
+    def search_pages(self, locale, search_query):
+        return BlogPage.objects.live().public().filter(locale=locale).search(search_query)
+
+    @route(r"^search/$")
+    def blog_page_search(self, request, *args, **kwargs):
+        search_query = request.GET.get("q", None)
+
+        if search_query:
+            self.search_term = search_query
+            self.search_type = 'search'
```

Comment:
**Question:** What other values might the `search_type` have?

Reply:
ah, good point that line should not have been there anymore. It'd be useful for things like /tags or /topics for the future blog work, but not really relevant for RH